index.js

```diff
@@ -2,25 +2,38 @@
 const { ProvidePlugin } = require("webpack")
 const filterObject = require("filter-obj")
 
-const excludeObjectKeys = (object, excludeKeys) => filterObject(object, key => !excludeKeys.includes(key))
+function createAliasFilter({ includeAliases, excludeAliases }) {
+	if (includeAliases.length > 0) {
+		return object => filterObject(object, key => includeAliases.includes(key))
+	}
+
+	return object => filterObject(object, key => !excludeAliases.includes(key))
+}
 
 module.exports = class NodePolyfillPlugin {
 	constructor(options = {}) {
 		this.options = {
 			excludeAliases: [],
+			includeAliases: [],
 			...options
 		}
+
+		if (this.options.includeAliases.length > 0 && this.options.excludeAliases.length > 0) {
+			throw new Error("excludeAliases and includeAliases are mutually exclusive!")
```

Comment:
Changed my mind on this

```suggestion
			throw new Error("excludeAliases and includeAliases are mutually exclusive")
```